monero-outreach-docs/translations/zh-hans/website-sections/homepage_zh-hans.md

```diff
@@ -0,0 +1 @@
+Homepage  www.monerooutreach.org

1. Monero Outreach：  门罗币推广组
2. Stories : 故事
3. Videos: 视频
4. Gallery : 画廊
5. Resources: 资源
6. On guard: 警惕
7. Every hash helps secure the network:  每一个哈希都保护着整个网络。
8. Mine Monero to Support the Network: 参与门罗币挖矿，支持门罗币网络建设。
9. Join the fight to secure private cash for all!  加入门罗币战队，保护每个人的私人现金。
10. Here are some practical options that you might consider to help us secure the network.
您可以从以下实用选项中考虑帮助我们保护网络。
11. Monero Best Practices  门罗币最佳实践
12. Investor  投资者
13. About us  关于我们

```

Comment:
Hey @al-mutasim one thing I forgot about this file. Could please change the name to _**"index_zh-hans.md"**_.